lib/tasks/forms_admin.rake

```diff
@@ -0,0 +1,14 @@
+namespace :forms_admin do
+  desc "Unlive a form"
+  task :make_unlive, [:form_id] => :environment do |_, args|
+    form = Form.find(args[:form_id])
+    form.make_unlive!
+    form.reload
+
+    if form.has_live_version
```

Comment:
I think I'd expect `form.has_live_version` to always return `false` if we've already run `form.make_unlive!` in the task - if it doesn't then I feel like it might not be checking the right thing

Reply:
```ruby
def has_live_version
  made_live_forms.present?
end
```
I think that'll return `true` if `made_live_forms` is `[]`, which I think might be what `made_live_forms` returns when we've run `made_live_forms.destroy_all`

Reply:
So the `else` statement here won't get much action, and I think that would also explain why you're not seeing `Form has no live version` when running `make_unlive` on a form that's already been un-alived

Reply:
Hmmm but `.present?` should do:
```ruby
irb(main):001> [].present?
=> false
```

Reply:
Ah, you're right. I'm now wondering what made_live_forms is returning then!

Reply:
Ahhhhhhh!!!!  😖 
```ruby
irb(main):001> form = Form.find(20)
irb(main):004> form.has_live_version
=> false
```

Reply:
It took me far to long to see that the logic was in the wrong place 🥲